src/style.scss

```diff
@@ -28,19 +28,63 @@
     @include unbuttonize;
 
     cursor: pointer;
-    color: var(--auro-color-text-primary-on-light);
+    display: flex;
+    color: var(--auro-color-text-primary-default);
```

Comment:
Let's look at just try doing the change to `--ds-...` now.

Reply:
There is still an issue when implementing the `--ds` prefix for the tokens. When implementing the new design token work, the styles would not render. Let's wait on this change so we can get this work out.